### PR TITLE
Use black version in REFLO environment for GHA

### PR DESCRIPTION
**Pull request by kurbansitterley**
Edit checks.yml to use the black version specified in the environment build when doing linting test with GHA.

The modified lines in checks.yml are taken directly from checks.yml in main watertap.

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -12,13 +12,24 @@ env:
 jobs:
 
   linting:
+    name: Check code is formatted (Black)
+    # OS and/or Python version don't make a difference, so we choose ubuntu and 3.8 as defaults
```
Comment (lbianchi-lbl) [inline .github/workflows/checks.yml:16]:
```suggestion
    # OS and/or Python version don't make a difference, so we choose ubuntu and 3.10 as defaults
```

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -12,13 +12,24 @@ env:
...
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: psf/black@stable
+      - uses: actions/setup-python@v5
         with:
-          options: "--check --verbose"
-          version: "~= 22.0"
+          python-version: '3.8'
```
Comment (lbianchi-lbl) [inline .github/workflows/checks.yml:22]:
```suggestion
          python-version: '3.10'
```

File: .github/workflows/checks.yml [hunk 3]
```diff
@@ -12,13 +12,24 @@ env:
...
+      - name: Install Black
+        # unlike the other jobs, we don't need to install WaterTAP and/or all the dev dependencies,
```
Comment (lbianchi-lbl) [inline .github/workflows/checks.yml:24]:
```suggestion
        # unlike the other jobs, we don't need to install watertap-reflo and/or all the dev dependencies,
```

**Review by lbianchi-lbl — COMMENTED** [review]
Looks good. I left some a couple of suggestion to update the Python version from 3.8 (unsupported by the latest IDAES even if in this case it doesn't technically matter), and fix a reference to WaterTAP that was left in another comment.

**Review by lbianchi-lbl — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]

**Pull request merged**